constants_local.xml

```diff
@@ -208,6 +209,7 @@
     <transactions>
         <TOTAL_COINBASE_REWARD>8400000000000000000000</TOTAL_COINBASE_REWARD>
         <COINBASE_REWARD_PER_DS>191780820000000000</COINBASE_REWARD_PER_DS>
+        <TOTAL_GENESIS_TOKEN>12600000000000000000000</TOTAL_GENESIS_TOKEN>
```

Comment:
shall we add xml comment saying the unit is in `Qa` and not `zil` to avoid confusion

Reply:
I think all of our unit for balance in protocol side are `Qa` intentionally